airbyte-config/init/src/main/resources/seed/destination_specs.yaml

```diff
@@ -293,10 +293,10 @@
           type: "object"
           title: "Loading Method"
           description: "Loading method used to send select the way data will be uploaded\
-            \ to BigQuery. <br><b>Standard Inserts</b> - Direct uploading using SQL\
+            \ to BigQuery. <br/><b>Standard Inserts</b> - Direct uploading using SQL\
```

Comment:
curious, docusarus doesn't like this formatting?

Reply:
JSX expects a closed tag and docusaurus uses JSX building static markdown.  So yes